content/js1/sprints/1/prep/index.md

```diff
@@ -8,4 +8,443 @@ backlog= 'Module-JS1'
+
+Now we can refer to the label `greeting` again in the REPL:
+
+![reference-greeting](reference-greeting.png)
+
+Now we have the `greeting` variable stored in memory we can reuse it to build more expressions:
+
+![use-greeting-variable](use-greeting-variable.png)
+
+In the example above, we're using backticks to create a template literal. With template literals, we can insert expressions into strings to produce new strings. Any time we want to reference a variable inside a template literal we use a dollar sign and a set of curly braces. The variable name itself is then placed inside the curly braces. The value this variable holds is then placed inside the string.
+
+### Reusing instructions
+
+Let’s consider another scenario. Instead of adding or multiplying numbers, we’ll consider a number like `10.3`.  
+Given the number `10.3`, we could ask:
+
+> 🤔 "What is the nearest whole number to `10.3`?"
+
+The process of finding the nearest whole number to 10.3 is called **rounding**. So we could rephrase the previous question to be:
+
+> 🤔 "What does the number `10.3` **round** to?”
+
+Again we can use the Node REPL to round the number 10.3 to the nearest whole number. However, in this case, there is no arithmetical operator for rounding the number `10.3` in JavaScript.
+
+However, we will often want to carry out some sequence of operations **again and again**.
+For example, we will often want to round numbers again and again.
+
+So in this case we can use a **function**.
+
+{{<note title="Definition" type="info">}}
+
+A function is a reusable set of instructions.
+
+{{</note>}}
+
+`Math.round` is an example of a function. Recalling the definition, a function is a reusable set of instructions: `Math.round` contains instructions for rounding any number.
+
+Functions will often take **inputs** and then **apply their set of instructions to the inputs** to produce an **output**.
+
+So we can write `Math.round` in the Node REPL:
+
+![round-img](math.round-img.png)
+
+After we've hit enter to evaluate, we then get the following:
+
+![round-img](math.round-evaluated.png)
+
+So in this case, the REPL output `[Function: round]` is indicating that `Math.round` is a function.
+
+### 📲 Calling a function
+
+Once we have the name for a function, we need to get Node to read the function's instructions and execute them. Execution means the computer reads the instructions and carries them out. So we can write the following in the REPL:
+
+![round-img](math.round-number-10.3.png)
+
+Notice now we have `(` and `)` parentheses after the name of the function and a number inside the parentheses. The parentheses indicate that we are calling the function. The number inside the parentheses is the **input** we're passing to the function.
+
+{{<note title="Definition" type="info">}}
+
+**Calling a function** means telling the computer to read the function's instructions and carry out its instructions. When calling a function we can also pass inputs to the function.
```

Comment:
Would it make sense to introduce here the synonyms for "calling a function"? Invoking a function, executing a function... any others?